apps/entity/api.py

```diff
@@ -157,7 +158,12 @@ def put(self, request, data=None, allow_partial=False, **kwargs):
         serializer.is_valid()
         errors = serializer.errors
         if errors:
-            fields = {"error_message": "my_error_message", "error_code": 0}
+            print(errors)
+            fields = {}
+            for key, value in errors.items():
+                error_code = get_form_error_code(vars(value[0])['code'])
+                fields[key] = {"error_message": f"{value[0]}", "error_code": error_code}
```

Comment:
Een attribuut kan meerdere errors hebben, we willen dus niet alleen de eerste error (`value[0]`) maar de hele lijst teruggeven:

```
errors = [
  name: [ { message: 'is required', code: 1 } ]
]
```

Ook mogen alle `prints` er nog uit.